packages/babel-preset-env/src/targets-parser.js

```diff
@@ -194,35 +177,36 @@ const getTargets = (targets: Object = {}, options: Object = {}): Targets => {
   }
 
   // Parse browsers target via browserslist
-  const browsersquery = validateBrowsers(targets.browsers);
+  let browsersquery = validateBrowsers(targets.browsers);
 
   const hasTargets = Object.keys(targets).length > 0;
   const shouldParseBrowsers = !!targets.browsers;
   const shouldSearchForConfig =
     !options.ignoreBrowserslistConfig && !hasTargets;
 
-  if (shouldParseBrowsers || shouldSearchForConfig) {
-    // If no targets are passed, we need to overwrite browserslist's defaults
-    // so that we enable all transforms (acting like the now deprecated
-    // preset-latest).
-    //
-    // Note, if browserslist resolves the config (ex. package.json), then usage
-    // of `defaults` in queries will be different since we don't want to break
-    // the behavior of "no targets is the same as preset-latest".
-    if (!hasTargets) {
```

Comment:
When `hasTargets` is `false`, `objectToBrowserslist(targets)` always return `[]`.

Reply:
Is there any difference between `[]` and `["defaults"]`?